src/parse/rules/cc_rules.build_defs

```diff
@@ -495,6 +497,17 @@ def cc_test(name:str, srcs:list=[], hdrs:list=[], compiler_flags:list&cflags&cop
             _c=_c,
         )
         deps.append(lib_rule)
+
+    test_cmd = '$TEST' % flags
```

Comment:
this is missing any symbol to interpolate?

Reply:
oops! Will fix this now!